test/interop/fortran/multidimArray/chapelProcs.chpl

```diff
@@ -0,0 +1,53 @@
+use ISO_Fortran_binding;
+
+proc toChapelArray(ref A: CFI_cdesc_t, param rank = 1) {
+  assert(A.rank == rank);
+  var D: domain(rank);
+  var dims: rank*range;
+
+  for i in 0..#rank {
+    dims(i+1) = A.dim[i].lower_bound..A.dim[i].extent /* by A.dim[i].sm ? */;
+  }
+
+  D = {(...dims)};
+  var AA: [D] int; // getChplType(A.type);
```

Comment:
I'm assuming this note is for when the support is first class instead of a prototype?

Reply:
I'm not sure how to write that `getChplType()` method.  We'd need to somehow convert the runtime value in `A.type` into a compile-time type. Maybe we can do some trickery to allow the type to be found at runtime though.

Reply:
Ah, it hadn't clicked for me that this would have to be a runtime value.  That makes sense.